=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.msc.service.ServiceContainerImpl <clinit>
INFO: JBoss MSC version 1.4.5.Final
Oct 03, 2018 4:56:15 PM org.jboss.threads.Version <clinit>
INFO: JBoss Threads version 2.3.2.Final
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "response-headers" => {"runtime-update-skipped" => true},
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => 2,
            "rolled-back" => true
        }
    },
    "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
    "rolled-back" => true
}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("missing-service") failed - address: ([]) - failure description: {
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.missing"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => ["jboss.missing-service is missing [jboss.missing]"]
}
======================
=========  New Test 

{
    "outcome" => "success",
    "result" => {
        "step-1" => {
            "outcome" => "success",
            "result" => 1,
            "response-headers" => {
                "runtime-update-skipped" => true,
                "operation-requires-reload" => true
            }
        },
        "step-2" => {
            "outcome" => "success",
            "result" => 2
        }
    },
    "response-headers" => {"process-state" => "reload-required"}
}
======================
=========  New Test 

======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.msc.service.ServiceControllerImpl startFailed
ERROR: MSC000001: Failed to start service jboss.bad-service
org.jboss.msc.service.StartException in service jboss.bad-service: Failed to start service
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1730)
	at org.jboss.msc.service.ServiceControllerImpl$ControllerTask.run(ServiceControllerImpl.java:1558)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1985)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1487)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1378)
	at java.lang.Thread.run(Thread.java:811)
Caused by: java.lang.RuntimeException: Bad service!
	at org.jboss.as.controller.ModelControllerImplUnitTestCase$BadServiceHandler$1$1.start(ModelControllerImplUnitTestCase.java:1112)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1738)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1700)
	... 6 more

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("bad-service") failed - address: ([]) - failure description: {"WFLYCTL0080: Failed services" => {"jboss.bad-service" => "Failed to start service
    Caused by: java.lang.RuntimeException: Bad service!"}}
======================
=========  New Test 

======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => 2,
            "rolled-back" => true
        }
    },
    "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
    "rolled-back" => true
}
======================
=========  New Test 

======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "failure-description" => "this request is bad",
            "rolled-back" => true
        }
    },
    "failure-description" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-2" => "this request is bad"}},
    "rolled-back" => true
}
======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "failure-description" => "this request is bad",
            "rolled-back" => true
        }
    },
    "failure-description" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-2" => "this request is bad"}},
    "rolled-back" => true
}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("handleFailed") failed - address: ([]) - failure description: "handleFailed"
======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "response-headers" => {"runtime-update-skipped" => true},
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => 2,
            "rolled-back" => true
        }
    },
    "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
    "rolled-back" => true
}
======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => {
                "step-1" => {
                    "outcome" => "failed",
                    "result" => 1,
                    "rolled-back" => true
                },
                "step-2" => {
                    "outcome" => "failed",
                    "result" => 2,
                    "rolled-back" => true
                }
            },
            "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => {
                "step-1" => {
                    "outcome" => "failed",
                    "result" => 2,
                    "rolled-back" => true
                },
                "step-2" => {
                    "outcome" => "failed",
                    "result" => 1,
                    "rolled-back" => true
                }
            },
            "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
            "rolled-back" => true
        }
    },
    "failure-description" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {
        "Operation step-1" => "WFLYCTL0063: Composite operation was rolled back",
        "Operation step-2" => "WFLYCTL0063: Composite operation was rolled back"
    }},
    "rolled-back" => true
}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.msc.service.ServiceControllerImpl startFailed
ERROR: MSC000001: Failed to start service jboss.bad-service
org.jboss.msc.service.StartException in service jboss.bad-service: Failed to start service
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1730)
	at org.jboss.msc.service.ServiceControllerImpl$ControllerTask.run(ServiceControllerImpl.java:1558)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1985)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1487)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1378)
	at java.lang.Thread.run(Thread.java:811)
Caused by: java.lang.RuntimeException: Bad service!
	at org.jboss.as.controller.ModelControllerImplUnitTestCase$BadServiceHandler$1$1.start(ModelControllerImplUnitTestCase.java:1112)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1738)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1700)
	... 6 more

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("bad-service") failed - address: ([]) - failure description: {"WFLYCTL0080: Failed services" => {"jboss.bad-service" => "Failed to start service
    Caused by: java.lang.RuntimeException: Bad service!"}}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("evil") failed - address: ([])
java.lang.RuntimeException: this handler is evil
	at org.jboss.as.controller.ModelControllerImplUnitTestCase$ModelStageThrowsExceptionHandler.execute(ModelControllerImplUnitTestCase.java:923)
	at org.jboss.as.controller.AbstractOperationContext.executeStep(AbstractOperationContext.java:999)
	at org.jboss.as.controller.AbstractOperationContext.processStages(AbstractOperationContext.java:743)
	at org.jboss.as.controller.AbstractOperationContext.executeOperation(AbstractOperationContext.java:467)
	at org.jboss.as.controller.OperationContextImpl.executeOperation(OperationContextImpl.java:1411)
	at org.jboss.as.controller.ModelControllerImpl.internalExecute(ModelControllerImpl.java:423)
	at org.jboss.as.controller.ModelControllerImpl.lambda$execute$0(ModelControllerImpl.java:227)
	at org.jboss.as.controller.ModelControllerImpl$$Lambda$37.0000000098CAAF90.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity$$Lambda$38.00000000986F63B0.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:289)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:255)
	at org.jboss.as.controller.ModelControllerImpl.execute(ModelControllerImpl.java:227)
	at org.jboss.as.controller.CompositeOperationHandlerUnitTestCase.testModelStageUnhandledFailureNoRollback(CompositeOperationHandlerUnitTestCase.java:157)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "failure-description" => "WFLYCTL0158: Operation handler failed: java.lang.RuntimeException: this handler is evil",
            "rolled-back" => true
        }
    },
    "failure-description" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-2" => "WFLYCTL0158: Operation handler failed: java.lang.RuntimeException: this handler is evil"}},
    "rolled-back" => true
}
======================
=========  New Test 

{
    "outcome" => "success",
    "result" => {
        "step-1" => {
            "outcome" => "success",
            "result" => 1
        },
        "step-2" => {
            "outcome" => "success",
            "result" => 2
        }
    }
}
======================
=========  New Test 

{
    "outcome" => "success",
    "result" => {
        "step-1" => {
            "outcome" => "success",
            "result" => 1,
            "response-headers" => {
                "runtime-update-skipped" => true,
                "operation-requires-restart" => true
            }
        },
        "step-2" => {
            "outcome" => "success",
            "result" => 2
        }
    },
    "response-headers" => {"process-state" => "restart-required"}
}
======================
=========  New Test 

------
------
======================
=========  New Test 

======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("operationFailedException") failed - address: ([]) - failure description: "OFE"
{
    "outcome" => "success",
    "result" => {
        "step-1" => {
            "outcome" => "success",
            "result" => 1
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => 2,
            "failure-description" => "OFE",
            "rolled-back" => false
        }
    }
}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("handleFailed") failed - address: ([]) - failure description: "handleFailed"
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("runtimeException") failed - address: ([])
java.lang.RuntimeException: runtime exception
	at org.jboss.as.controller.ModelControllerImplUnitTestCase$RuntimeStageThrowsExceptionHandler$1.execute(ModelControllerImplUnitTestCase.java:988)
	at org.jboss.as.controller.AbstractOperationContext.executeStep(AbstractOperationContext.java:999)
	at org.jboss.as.controller.AbstractOperationContext.processStages(AbstractOperationContext.java:743)
	at org.jboss.as.controller.AbstractOperationContext.executeOperation(AbstractOperationContext.java:467)
	at org.jboss.as.controller.OperationContextImpl.executeOperation(OperationContextImpl.java:1411)
	at org.jboss.as.controller.ModelControllerImpl.internalExecute(ModelControllerImpl.java:423)
	at org.jboss.as.controller.ModelControllerImpl.lambda$execute$0(ModelControllerImpl.java:227)
	at org.jboss.as.controller.ModelControllerImpl$$Lambda$37.0000000098CAAF90.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity$$Lambda$38.00000000986F63B0.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:289)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:255)
	at org.jboss.as.controller.ModelControllerImpl.execute(ModelControllerImpl.java:227)
	at org.jboss.as.controller.CompositeOperationHandlerUnitTestCase.testRuntimeStageUnhandledFailureNoRollback(CompositeOperationHandlerUnitTestCase.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "result" => 1,
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "result" => 2,
            "failure-description" => "WFLYCTL0158: Operation handler failed: java.lang.RuntimeException: runtime exception",
            "rolled-back" => true
        }
    },
    "failure-description" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-2" => "WFLYCTL0158: Operation handler failed: java.lang.RuntimeException: runtime exception"}},
    "rolled-back" => true
}
======================
=========  New Test 

{
    "outcome" => "failed",
    "result" => {"step-1" => {
        "outcome" => "failed",
        "result" => 2,
        "rolled-back" => true
    }},
    "failure-description" => "WFLYCTL0063: Composite operation was rolled back",
    "rolled-back" => true
}
======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("evil") failed - address: ([])
java.lang.RuntimeException: this handler is evil
	at org.jboss.as.controller.ModelControllerImplUnitTestCase$ModelStageThrowsExceptionHandler.execute(ModelControllerImplUnitTestCase.java:923)
	at org.jboss.as.controller.AbstractOperationContext.executeStep(AbstractOperationContext.java:999)
	at org.jboss.as.controller.AbstractOperationContext.processStages(AbstractOperationContext.java:743)
	at org.jboss.as.controller.AbstractOperationContext.executeOperation(AbstractOperationContext.java:467)
	at org.jboss.as.controller.OperationContextImpl.executeOperation(OperationContextImpl.java:1411)
	at org.jboss.as.controller.ModelControllerImpl.internalExecute(ModelControllerImpl.java:423)
	at org.jboss.as.controller.ModelControllerImpl.lambda$execute$0(ModelControllerImpl.java:227)
	at org.jboss.as.controller.ModelControllerImpl$$Lambda$37.0000000098CAAF90.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity$$Lambda$38.00000000986F63B0.run(Unknown Source)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:289)
	at org.wildfly.security.auth.server.SecurityIdentity.runAs(SecurityIdentity.java:255)
	at org.jboss.as.controller.ModelControllerImpl.execute(ModelControllerImpl.java:227)
	at org.jboss.as.controller.CompositeOperationHandlerUnitTestCase.testModelStageUnhandledFailure(CompositeOperationHandlerUnitTestCase.java:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

======================
=========  New Test 

Oct 03, 2018 4:56:15 PM org.jboss.as.controller.AbstractOperationContext executeStep
ERROR: WFLYCTL0013: Operation ("missing-service") failed - address: ([]) - failure description: {
    "WFLYCTL0412: Required services that are not installed:" => ["jboss.missing"],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => ["jboss.missing-service is missing [jboss.missing]"]
}
======================
